mm2src/coins/qrc20.rs

```diff
@@ -1491,12 +1492,16 @@ pub struct Qrc20FeeDetails {
 }
 
 async fn qrc20_withdraw(coin: Qrc20Coin, req: WithdrawRequest) -> WithdrawResult {
-    let to_addr = UtxoAddress::from_str(&req.to)
-        .map_err(|e| e.to_string())
-        .map_to_mm(WithdrawError::InvalidAddress)?;
+    let to_addr = UtxoAddress::from_legacyaddress(
+        &req.to,
+        coin.as_ref().conf.pub_addr_prefix,
+        coin.as_ref().conf.pub_t_addr_prefix,
+        coin.as_ref().conf.p2sh_addr_prefix,
+        coin.as_ref().conf.p2sh_t_addr_prefix,
```

Comment:
What do you think about creating the following struct and using it instead to make code a bit shorted/avoid duplication?
```rust
struct AddressPrefixes {
    pub_addr_prefix: u8,
    pub_t_addr_prefix: u8,
    p2sh_addr_prefix: u8,
    p2sh_t_addr_prefix: u8,
}
```
So this and other `from_legacyaddress` calls will look like
```rust
UtxoAddress::from_legacyaddress(&req.to, coin.as_ref().conf.address_prefixes)
```

Reply:
yes, I also thought about a struct like that or array. I will do this, thank you

Reply:
What would you say about using Vec<u8> for prefixes, like it is in daemons codebases. So such a vec would look like [t_addr_prefix, p2pkh_prefix] with t_addr_prefix optional.
```
pub struct Address {
    /// The prefixes of the p2pkh address of size 1 or 2
    pub p2pkh_prefix: Vec<u8>,
    /// The prefixes of the p2sh address of size 1 or 2
    pub p2sh_prefix: Vec<u8>,
    ...
}
```
Functions would accept slices in their params.
I think it would be easier to deal with such types (less variables or members)
Also we do not have t_addr_prefix in most cases, so we do not process it with this approach.
Plus this is consistent with other prefixes representations like xpub

Reply:
Added NetworkAddressPrefixes and AddressPrefixes structs to encapsulate prefixes